src/instructlab/cli/model/evaluate.py

```diff
@@ -35,15 +35,13 @@
 )
 @click.option(
     "--judge-model",
+    default=None,
     type=click.STRING,
-    cls=clickext.ConfigOption,
-    config_sections="mt_bench",
 )
 @click.option(
     "--output-dir",
+    default=None,
```

Comment:
same comment as judge model

Reply:
After speaking to @RobotSail I think maybe the other option I mentioned below of filing a follow up issue to expand clickext to do this defaulting for us based on `--benchmark` is the better approach.

I generally am not a fan of expanding this complex click defaulting but there doesn't seem to be another easy solution here. 

Each dk-specific option though should have `--dk-...` and a default from the cfg.

Reply:
@cdoern The behavior of `--output-dir` right now could be polished. Namely, the way it should function would be as follows:

- defaults to `{ilab-storage-dir}/eval/{benchmark}`
- When specified, just go wherever is pointed to

This way if we wanted to implement `ilab eval list` in the present system, it's very easy to group data based on different evals.